README.md

```diff
@@ -420,7 +425,7 @@ Other Style Guides
     });
     ```
 
-<a name="arrays--bracket-newline"></a>
+  <a name="arrays--bracket-newline"></a>
```

Comment:
why this indentation change?

Reply:
Because other similar parts of this guide have this indentation. Actually, this indentations don't affect the markdown preview on Github, so if needed I can remove this change.